recipes/lora_finetune_distributed.py

```diff
@@ -545,15 +555,20 @@ def _setup_data(
 
         if isinstance(cfg_dataset, ListConfig):
             datasets = [
-                config.instantiate(single_cfg_dataset, tokenizer=self._tokenizer)
+                config.instantiate(single_cfg_dataset, self._tokenizer)
```

Comment:
This doesn't need to be keyword?

Reply:
Some datasets call it tokenizer and others call it transforms